### PR TITLE
Option -c to compare a list of files. Fixes #39

### LINKED ISSUE #39: Feature request: compare two files
Subj. It would be nice to have an option to launch jplag just to compare two files, without creating dirs, etc.

### PR DESCRIPTION
**Pull request by ov7a**

### CODE REVIEW

File: jplag/src/main/java/jplag/options/Options.java [hunk 1]
```diff
@@ -181,8 +189,8 @@ public abstract void initializeSecondStep(Program program)
 	public static void usage() {
 		System.out.print(Program.name_long
 						+ ", Copyright (c) 2004-2017 KIT - IPD Tichy, Guido Malpohl, and others.\n"
-						+ "Usage: JPlag [ options ] <root-dir>\n"
-						+ " <root-dir>        The root-directory that contains all submissions\n\n"
+						+ "Usage: JPlag [ options ] <root-dir> [-c file1 file2 ...]\n"
```
Comment (lama0206) [inline jplag/src/main/java/jplag/options/Options.java]: Having to have a root-dir _and_ the file list seems awkward. When running in the new mode, the file list should be enough.

File: README.md [hunk 2]
```diff
@@ -57,6 +58,10 @@ The option `-x` requires an exclusion list saved as `<file>`.
 The exclusion list contains a  number of suffixes.
 JPlag will ignore all files that end with one of the suffixes.
 
+### `-c [files]`   (Compare) Compare a list of files
+Example: `java -jar jplag-yourVersion.jar root_dir -l java17 -c student1_file student2_file student3_file`
+This option should be the last one.
```
Comment (lama0206) [inline README.md]: _must be_

File: jplag/src/main/java/jplag/options/Options.java (hunk 1 above)
Comment (ov7a) [reply jplag/src/main/java/jplag/options/Options.java]: I agree, but it was not very easy to eliminate it quickly, as it required in Submission class. I'll try to fix it this week.

File: README.md (hunk 2 above)
Comment (ov7a) [reply README.md]: Oops, my bad:(

File: jplag/src/main/java/jplag/options/Options.java (hunk 1 above)
Comment (ov7a) [reply jplag/src/main/java/jplag/options/Options.java]: It was easier than I expected. Seems that providing a null dir in submission does not break the main functions of jplag. Please, review my latest commit.

**Comment by ov7a** [comment]
@lama0206 Hi, any updates on this? Should I change something?

**Pull request merged**